src/Console/ConfigurationResolver.php

```diff
@@ -513,9 +513,13 @@ private function computeConfigFiles()
      */
     private function createFixerFactory()
     {
-        $fixerFactory = new FixerFactory();
-        $fixerFactory->registerBuiltInFixers();
-        $fixerFactory->registerCustomFixers($this->getConfig()->getCustomFixers());
+        static $fixerFactory = null;
```

Comment:
now it will be cached across different instances of `ConfigurationResolver`. very problematic for tests and using `PHP-CS-Fixer` as library. static is global, and then you base it on config, which may be different for different `ConfigurationResolver` instances